apputils/test/test_stemming_eng_logic.py

```diff
@@ -0,0 +1,76 @@
+import logging
+from unittest import TestCase
+
+from stemming import stemming
+
+logging.basicConfig(level = logging.INFO)
+logger = logging.getLogger('stemmingTest')
+
+
```

Comment:
for next time,each commit need to have 1 agenda.
you pushed 3 agenda with 1 commit

Reply:
will keep in mind next time